kong/tools/queue_schema.lua

```diff
@@ -0,0 +1,46 @@
+local Schema = require "kong.db.schema"
+
+-- TODO: enable the descriptions once they are accepted in Schemas
+return Schema.define {
+  type = "record",
+  fields = {
+    { name = {
+      type = "string",
+      -- description = "name of the queue, unique across one workspace.",
+      -- If two plugin instances use the same queue name, they will
+      -- share one queue and their queue related configuration must match.
+    } },
+    { max_batch_size = {
+      type = "number",
+      default = 1,
+      -- description = "maximum number of entries that can be processed at a time"
+    } },
+    { max_delay = {
+      type = "number",
+      default = 1,
+      -- description = "maximum number of (fractional) seconds to elapse after the first entry was queued before the queue starts calling the handler",
+      -- This parameter has no effect if `max_batch_size` is 1, as queued entries will be sent
+      -- immediately in that case.
+    } },
+    { max_entries = {
+      type = "number",
+      default = 10000,
+      -- description = "maximum number of entries that can be waiting on the queue",
+    } },
+    { max_bytes = {
```

Comment:
Some suggestions on better names:
- capacity_in_bytes
- capacity_bytes
- capcity_string_bytes

Reply:
The byte capacity parameter previously had `capacity` in the name but was changed in response to a review comment to unify them all into having the `max_` prefix.  I'm leaving it this way.